db/migrate/20230817135045_adds_primary_key_for_id_and_type.rb

```diff
@@ -0,0 +1,15 @@
+Sequel.migration do
+  up do
+    alter_table(:search_suggestions) do
+      drop_constraint(:search_suggestions_pkey, type: :primary_key)
+      add_primary_key [:id, :type]
+    end
+  end
+
+  down do
+    alter_table(:search_suggestions) do
+      drop_constraint(:search_suggestions_pkey, type: :primary_key)
```

Comment:
This is the same as line#4, is that right?

Reply:
No no, I'm changing from id, type back to id, value, here just in case I need to rollback